src/components/dls/Modal/Modal.tsx

```diff
@@ -15,10 +15,10 @@ const Modal = ({ children, trigger, open, onClickOutside, triggerClassName }: Mo
     <DialogPrimitive.Overlay className={styles.overlay} />
     {trigger && (
       <DialogPrimitive.Trigger
-        as="div"
+        asChild
```

Comment:
interesting, so with `asChild` it will not render this div? it will only render the child?